src/Core/Resolvers/Sql Query Structures/SqlQueryStructure.cs

```diff
@@ -59,15 +58,10 @@ public class SqlQueryStructure : BaseSqlQueryStructure
         /// </summary>
         public Dictionary<string, string> ColumnLabelToParam { get; }
 
-        /// <summary>
-        /// Default limit when no first param is specified for list queries
-        /// </summary>
-        private const uint DEFAULT_LIST_LIMIT = 100;
-
         /// <summary>
         /// The maximum number of results this query should return.
         /// </summary>
-        private uint? _limit = DEFAULT_LIST_LIMIT;
+        private uint? _limit = PaginationOptions.DEFAULT_PAGE_SIZE;
```

Comment:
its not really a breaking change right? as this is backward compatible. if user does not specifiy pagination options this will still be 100 as was before.

Reply:
Gotcha, Resolving because of:
> if user does not specifiy pagination options this will still be 100 as was before.